### PR TITLE
[fix] Restyled the pdf onboarding page

### PR DESCRIPTION
**Pull request by kevin3656**
### Description
[//]: # "Required - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."
This PR restyled the pdf files to match the figma more closely. This includes adding a title and pdf icon as well as buttons to navigate through the pdf

### Screenshots
[//]: # "Required for frontend changes, otherwise optional but strongly recommended. Add screenshots of expected behavior - GIFs if you're feeling fancy! Use the provided image template. Drag the desired image into the PR, then copy the link into the placeholder."

Screenshot 2025-04-27 at 2 42 24 PM

## How to review
[//]: # 'Required - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?'
The main files worked on were `onboarding/page.tsx` and `onboarding/styles.ts` the `pdfContainer` was added for borders and pdf title, and a the `pdfButtonsContainer` was reworked to match the figma. Can be tested by going through onboarding flow and testing buttons on pdf container navigate through pdf correctly

## Next steps
[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."

### Online sources
[//]: # 'Copy links to any tutorials or documentation that was useful to you when working on this PR'

### Related PRs
[//]: # "Add related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

CC: @kylezryr

### CODE REVIEW

File: app/onboarding/styles.ts [hunk 1]
```diff
@@ -108,3 +97,11 @@ export const StyledInput = styled(P1).attrs({ as: 'input' })<{
     outline: none;
   }
 `;
+
+export const PdfContainer = styled.div`
```
Comment (kylezryr) [inline app/onboarding/styles.ts:109]: image
i think we need a little bit of margin around the outside/put it in a container with padding because it looks like this on mobile

File: app/onboarding/page.tsx [hunk 2]
```diff
@@ -101,70 +103,80 @@ function PdfScreen({
...
-                          color: 'white',
-                          borderColor: 'rgba(200, 200, 200, 0.6)',
-                        }
-                  }
-                >
-                  Next
-                </Button>
-              </PDFButtonsContainer>
-            </PDFPageWrapper>
+            <PdfContainer>
```
Comment (kylezryr) [inline app/onboarding/page.tsx]: could we make the pdf page clickable and if you click it we open the PDF in a new tab? this is done in the guides tab in resources if you need a reference!

**Review by kylezryr — CHANGES_REQUESTED** [review]
looking really good! just some small changes to make and its ready to merge

**Review by kylezryr — APPROVED** [review]
ready 2 merge
